ApnsPHP/Message/Custom.php

```diff
@@ -33,6 +33,7 @@ class ApnsPHP_Message_Custom extends ApnsPHP_Message
 	protected $_aLocArgs; /**< @type array Variable string values to appear in place of the format specifiers in loc-key. */
 	protected $_sLaunchImage; /**< @type string The filename of an image file in the application bundle. */
 	protected $_sTitle; /**< @type string The title of an short looknotification displayed on Apple Watch. */
+    protected $_sSubTitle; /**< @type string The subtitle of a secondary description */
```

Comment:
Indentation. Also, iOS 10 uses `subtitle` instead of `subTitle`, so you might want to update the property and it's usages.